### PR TITLE
Humble support - Changes requested done

### PR DESCRIPTION
**Pull request by NicolasDubois0**
Hi ! 

I'm currently using humble and Husky ! I saw this pull request : https://github.com/husky/husky/pull/275
I made the required changes to allow the merge of the fixes on official repo.

### CODE REVIEW

File: husky_control/launch/teleop_base.launch.py [hunk 1]
```diff
@@ -14,14 +14,14 @@ def generate_launch_description():
...
 
-    node_interactive_marker_twist_server = Node(
-        package='interactive_marker_twist_server',
-        executable='marker_server',
-        name='twist_server_node',
-        remappings={('cmd_vel', 'twist_marker_server/cmd_vel')},
-        parameters=[filepath_config_interactive_markers],
-        output='screen',
-    )
+    # node_interactive_marker_twist_server = Node(
```
Comment (tonybaltovski) [inline husky_control/launch/teleop_base.launch.py]: This has been released for Humble, can you re-add it please?

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: husky_control/launch/teleop_base.launch.py (hunk 1 above)
Comment (NicolasDubois0) [reply husky_control/launch/teleop_base.launch.py]: Yes, I will re-add it

Comment (NicolasDubois0) [reply husky_control/launch/teleop_base.launch.py]: done

**Review by tonybaltovski — APPROVED** [review]

**Comment by tonybaltovski** [comment]
Looks good, I'll merge it but for Humbe, the Husky code will be moved to https://github.com/clearpathrobotics/clearpath_robot. Thanks!

**Pull request merged**